.github/ISSUE_TEMPLATE.md

```diff
@@ -1,34 +1,47 @@
-<!--- Provide a general summary of the issue in the Title above -->
-
-Your issue may already be reported!
-Please search on the [issue track](../) before creating one.
-
-## Expected Behavior
 <!--- If you're describing a bug, tell us what should happen -->
 <!--- If you're suggesting a change/improvement, tell us how it should work -->
+## Expected Behavior
```

Comment:
Is there a benefit to flipping the heading after the explanation of what to enter instead of before?  Just from my own experience, I've always highlighted the comments and typed over them with what it asks for.  Now it takes a little searching to figure out where that text should go after removing the comments.

I'm okay with trying it, but just something to consider.  Interested in what @jeroenptrs thinks too.

Reply:
Personally I found it annoying :P